--- a/packages/game/src/present/follower-pos.ts
+++ b/packages/game/src/present/follower-pos.ts
@@ -65,7 +65,7 @@
   let offX: number
   let offY: number
   if (m === 2) {
-    offX = baseDir === 'right' || baseDir === 'left' ? -16 : 16 // East||West ? -16 : +16
+    offX = 16 // East||West ? -16 : +16
     offY = 8
   } else {
     offX = baseDir === 'left' || baseDir === 'down' ? 16 : -16 // West||South ? +16 : -16
