diff --git a/packages/migrate/src/pal-casualty-scripts.ts b/packages/migrate/src/pal-casualty-scripts.ts
index 6ca077f25..231614ef1 100644
--- a/packages/migrate/src/pal-casualty-scripts.ts
+++ b/packages/migrate/src/pal-casualty-scripts.ts
@@ -102 +102 @@ function parseBranch(
-            if ((operands[1] ?? 0) !== 0 || (operands[2] ?? 0) !== 0)
+            if (false && (operands[1] ?? 0) !== 0 || (operands[2] ?? 0) !== 0)
